main.py

```diff
@@ -47,8 +61,11 @@ def transform_to_log_line(log_event):
     timestamp = log_event['timestamp']
     dt = datetime.utcfromtimestamp(timestamp / 1000.0)
     datetime_iso8601 = dt.isoformat() + 'Z'
+
     message = log_event['message']
+
     line = datetime_iso8601 + ": " + message
```

Comment:
@LucioFranco I added a task in the transformation 2.1 project to ensure we're parsing this format correctly. Do you think it makes sense to change this or support it?